docs/src/charm/howto/upgrade-minor.md

```diff
@@ -0,0 +1,216 @@
+  stable version of Juju (see the [Juju docs][juju-docs])
+* You read the [Upgrade notes][upgrade-notes] to see if any
+  caveats apply to the versions you are upgrading to/from
+* You read the [Upstream release notes][upstream-notes] for details
+  of Kubernetes deprecation notices and API changes that may impact
+  your workloads
+
+
+It is also important to understand that Kubernetes will only 
+upgrade and if necessary migrate, components relating specifically
+to elements of Kubernetes installed and configured as part of Kubernetes.
+This may not include any customized configuration of Kubernetes,
+or non-built-in generated objects (e.g. storage classes) or deployments which
+rely on deprecated APIs.
+
+## Specific upgrade instructions
+
+### Deciding if an upgrade is available
+
+Juju will contact [charmhub] daily to find new revisions of charms
+deployed in your models. To see if the `k8s` or `k8s-worker` charms 
+can be upgraded, set with the following:
+
+```sh
+juju status --format=json | \
+   jq '.applications | 
+        to_entries[] | {
+           application: .key,
+           "charm-name": .value["charm-name"],
+           "charm-channel": .value["charm-channel"],
+           "charm-rev": .value["charm-rev"],
+           "can-upgrade-to": .value["can-upgrade-to"]
+        }'
+```
+
+This outputs a list of applications in the model:
+* the name of the application (ex. `k8s`)
+* the charm used by the application (ex. `k8s`)
+* the kubernetes channel this charm follows (ex. `1.31/stable`)
+* the current charm revision  (ex. `1001`)
+* the next potential charm revision (ex. `ch:amd64/k8s-1002`)
+
+If the `can-upgrade-to` revision is `null`, you are at the most
+stable release in this channel and you should continue with the
+[Pre Upgrade Check](#the-pre-upgrade-check).
+
+Otherwise, continue with the [Upgrade Patch](upgrade-patch) instructions.
+
+
+```{caution} Only update the charm to the next minor version.
+If the current `charm-channel` is `1.31/stable`, it's critical
+to refresh to the `1.32/stable`. Skipping channels (eg 1.31 -> 1.33)
+will result in the units blocking and indicating they cannot upgrade.
+
+See Kubernetes' [Version Skew Policy][version-skew-policy]
+```
+
+### The pre-upgrade-check
+
+Before running an upgrade, check that the cluster is
+steady and ready for upgrade. The charm will perform checks 
+necessary to confirm the cluster is in safe working order before
+upgrading.
+
+```sh
+juju run k8s/leader pre-upgrade-check
+```
+
+If no error appears, the `pre-upgrade-check` completed successfully.
+
+### Refreshing charm applications
+
+#### Control Plane units (k8s)
+
+Following the `pre-upgrade-check` update the control-plane nodes.
+
+```sh
+juju refresh k8s --channel ${NEXT_CHANNEL}
+juju status k8s --watch 5s
+```
+
+The `refresh` command instructs the juju controller to follow a new
+charm channel related to the Kubernetes release and use the new charm
```

Comment:
This is quite a long sentence. Can we split into two? Suggestion:
The `refresh` command instructs the juju controller to follow a new
charm channel related to the Kubernetes release. It also instructs the juju controller to use the new charm revision of the application's channel to upgrade each unit.